python/src/LOLAVoronoi_doc.i.in

```diff
@@ -0,0 +1,161 @@
+%feature("docstring") OT::LOLAVoronoi
+"LOLA-Voronoi sequential design algorithm.
+
+This class implements the sequential design algorithm described in [crombecq2011]_ which allows one
+to sequentially generate new input samples to an existing design built from input/output samples.
+It combines an exploration criterion based on Voronoi tesselation to identify
+undersampled input design regions and an exploitation criterion based on a measure
+of the nonlinearity of the response surface.
+
+The input sample Voronoi cell relative volumes are written:
+
+.. math::
+
+    v(\vect{p}_i) = \int_{\vect{x} \in \Rset^d} 1_{i = \argmin_{j \in [1, N]} ||\vect{p}_j - \vect{x}||_2} \inputMeasure(\vect{x}) dx
+
+and are approximated using Monte Carlo with a number of samples set by the :class:`~openturns.ResourceMap` entry `LOLAVoronoi-DefaultVoronoiSamplingSize`.
+
+For the nonlinearity criterion the algorithm only uses the data available and chooses a
+set of neighbouring samples :math:`N(\vect{p}_i) = {\vect{p}_{i1}, \hdots, \vect{p}_{im}}`
+with :math:`m = 2 d` to build the gradient approximation :math:`\vect{J}`.
+
+The least-squares method to compute the gradient is set by the :class:`~openturns.ResourceMap` entry `LOLAVoronoi-DecompositionMethod`.
```

Comment:
I suggest to move this line below, where the other ResourceMap keys are introduced.